src/NHibernate/Tool/hbm2ddl/SchemaUpdate.cs

```diff
@@ -141,7 +141,8 @@ public void Execute(Action<string> scriptAction, bool doUpdate)
 			autoKeyWordsImport = autoKeyWordsImport.ToLowerInvariant();
 			if (autoKeyWordsImport == Hbm2DDLKeyWords.AutoQuote)
 			{
-				SchemaMetadataUpdater.QuoteTableAndColumns(configuration);
+				SchemaMetadataUpdater.Update(configuration, dialect);
```

Comment:
I'm not a big fan of doing remote IO in a constructor.  There will be no improvement in this case because everywhere `Execute` is called, it is immediately after `new`ing  up a new `SchemaUpdate` instance.